docs/source/quickstart.md

```diff
@@ -1,88 +1,271 @@
 # Quickstart
 
-## How does it work?
+TRL is a comprehensive library for post-training foundation models using techniques like Supervised Fine-Tuning (SFT), Direct Preference Optimization (DPO), and Group Relative Policy Optimization (GRPO).
 
-Fine-tuning a language model via PPO consists of roughly three steps:
+## Installation
```

Comment:
this is covered in the "Installation" section, you can remove it

<img width="529" height="455" alt="Screenshot 2025-08-12 at 12 03 14 PM" src="https://github.com/user-attachments/assets/044c4932-6ad0-4130-89cb-9df5f67ffc6c" />